template/template_example.Rmd

```diff
@@ -0,0 +1,206 @@
+
+## Obtain the `.Rmd` file of this analysis
+
+To run this example yourself, [download the `.Rmd` or this analysis by clicking this link](https://alexslemonade.github.io/refinebio-examples/ <<RELATIVE PATH TO THIS ANALYSIS>> ). 
+You can open this `.Rmd` file in RStudio and follow the rest of these steps from there (See our [section about getting started with R notebooks](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-and-use-rmds) if you are unfamiliar with `.Rmd` files).
+Clicking this link will most likely send this to your downloads folder on your computer. 
+Move this `.Rmd` file to where you would like this example and its files to be stored.
+
+## Set up your analysis folders
+
+Good file organization is helpful for keeping your data analysis project on track!
+We have set up some code that will automatically set up a folder structure for you. 
+Run this next chunk to set up your folders! 
+
+If you have trouble running this chunk, see our [introduction to using Rmds](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-and-use-rmds) for more resources and explanations. 
+
+```{r}
+# Define the file path to the results directory
+data_dir <- "data" # Replace with path to desired data directory
+
+# Create the data folder if it doesn't exist
+if (!dir.exists(data_dir)) {
+  dir.create(data_dir)
+}
+
+# Define the file path to the plots directory
+plots_dir <- "plots" # Can replace with path to desired output plots directory
+
+# Create the plots folder if it doesn't exist
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+
+# Define the file path to the results directory
+results_dir <- "results" # Can replace with path to desired output results directory
+
+# Create the results folder if it doesn't exist
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+```
+
+In the same place you put this `.Rmd` file, you should now have three new empty folders called `data`, `plots`, and `results`!
+
+## Obtain the dataset from refine.bio
+
+For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-the-data). 
+
+Go to this [dataset's page on refine.bio](<<REFINEBIO DATASET PAGE LINK>>).
+
+Click the "Download Now" button on the right side of this screen.  
+
+<img src="screenshots/download-now.png" width=200>  
+
+Fill out the pop up window with your email and our Terms and Conditions:  
+
+<img src="screenshots/download-email.png" width=500>  
+
+It make take a few minutes for the dataset to process.
+You will get an email when it is ready. 
+
+## About the dataset we are using for this example
+
+For this example analysis, we will use this [<<BRIEF DATASET NAME>>](<<RELATIVE PATH TO THIS ANALYSIS>>).
```

Comment:
Are you expecting one to replace `<<RELATIVE PATH TO THIS ANALYSIS>>` with the path to the downloaded dataset on a given user's machine (which would vary)? Or is it the path that we will tell them to put the file in in the next section?

Reply:
I'll make this instruction more clear. But I'm just talking about the relative path within the refinebio-examples repo. This is the only part that will be relevant to the github.io link. E.g. `02-microarray/clusterin/clustering.html`